verifier/src/qe_identity.rs

```diff
@@ -80,20 +142,68 @@ impl<'a> TryFrom<&SignedQeIdentity<'a>> for QeIdentity {
     }
 }
 
-#[derive(Debug, Deserialize)]
+#[derive(Debug, Deserialize, Clone, PartialEq)]
 #[serde(rename_all = "camelCase")]
-struct TcbLevel {
+pub struct TcbLevel {
     tcb: Tcb,
     tcb_date: String,
     tcb_status: AdvisoryStatus,
     #[serde(rename = "advisoryIDs", default)]
     advisory_ids: Vec<String>,
 }
 
-#[derive(Debug, Deserialize)]
-struct Tcb {
+impl TcbLevel {
+    pub fn new<'a, I, E>(tcb: Tcb, tcb_status: AdvisoryStatus, advisory_ids: I) -> Self
+    where
+        I: IntoIterator<Item = &'a E>,
+        E: ToString + 'a + ?Sized,
+    {
+        Self {
+            tcb,
+            tcb_date: UNIX_TIME_STR.to_string(),
+            tcb_status,
+            advisory_ids: advisory_ids.into_iter().map(ToString::to_string).collect(),
+        }
+    }
+
+    pub fn isv_svn(&self) -> IsvSvn {
+        self.tcb.isv_svn.into()
+    }
+
+    pub fn status(&self) -> AdvisoryStatus {
+        self.tcb_status
+    }
+
+    pub fn advisory_ids(&self) -> &[String] {
+        &self.advisory_ids
+    }
```

Comment:
I considered having this return `Advisories`, then doing using the `is_superset()` method in the comparison logic. Up for suggestions here, as I think there was some concern with the `is_superset()` intuitiveness to consumers.

Reply:
Sorry I left too much context in my head.
After doing a  quick prototype, I went ahead and updated to use `Advisories` and the `AdvisoriesVerifier` here in https://github.com/mobilecoinfoundation/attestation/pull/114/commits/a80ec2b4702c058ead7fba41d062dc2dc27198b7